terraform/ops/main.tf

```diff
@@ -138,6 +138,19 @@ resource "google_artifact_registry_repository" "e2e_runner_docker" {
   ]
 }
 
+resource "google_artifact_registry_repository" "e2e_deployer_docker" {
```

Comment:
I'm trying to use something like a 1:1:* ratio of `artifact repository:component:container`, could we reduce to a single artifact repository, and rename:

```
${_REGION}-docker.pkg.dev/${PROJECT_ID}/e2e-testing/runner:${_IMAGE_TAG}
${_REGION}-docker.pkg.dev/${PROJECT_ID}/e2e-testing/deployer:${_IMAGE_TAG}
```